db.js

```diff
@@ -1,22 +1,25 @@
 var mysql = require("mysql2");
```

Comment:
Most of the changes in this file can be ignored. It's all prettier reformatting.

Reply:
Find my comment in this file below for the only real changes